src/components/Ecoverse/InterestModal.tsx

```diff
@@ -127,7 +127,7 @@ const InterestModal: FC<P> = ({ onHide, show, opportunityId }) => {
         </Grid>
       </DialogContent>
       <DialogActions>
-        {data?.createRelation.id && <Button onClick={onHide} variant={'primary'} text={t('buttons.cancel')} />}
+        {data?.createRelation.id && <Button onClick={onHide} variant={'primary'} text={t('buttons.close')} />}
```

Comment:
if this condition is not met the dialog is left unclosable.
Add `onClose` handler on the title component or make sure the dialog is closable somehow.
There was an issue for that.

Reply:
This wasn't part of the issue. Just `cancel` sounded too awkward.